src/commands/cni/test/flow.ts

```diff
@@ -0,0 +1,154 @@
+import { Flags, ux } from "@oclif/core";
+import axios from "axios";
+import inquirer from "inquirer";
+import { PrismaticBaseCommand } from "../../../baseCommand.js";
+import { listIntegrationFlows } from "../../../utils/integration/flows.js";
+import { exists, fs } from "../../../fs.js";
+import { getPrismMetadata } from "../../../utils/integration/metadata.js";
+
+const MISSING_PARAM_MESSAGE = "You must provide either a flow-url or an integration-id parameter.";
+
+export default class CniTestFlowCommand extends PrismaticBaseCommand {
+  static description = "Run a test execution of a CNI flow";
+
+  static flags = {
+    "flow-url": Flags.string({
+      char: "u",
+      description: "Invocation URL of the flow to run",
+      required: false,
+    }),
+    "integration-id": Flags.string({
+      char: "i",
+      description: "ID of the integration that the flow belongs to",
+      required: false,
+    }),
+    sync: Flags.boolean({
+      char: "s",
+      description: "Forces the flow to run synchronously",
+      required: false,
+    }),
+    "include-execution-id": Flags.boolean({
+      char: "e",
+      description: "Include the execution ID in the response of a synchronous flow execution",
+      required: false,
+    }),
+    "trigger-payload-file": Flags.string({
+      char: "p",
+      description: "A file containing a custom trigger payload to run the flow with",
+      required: false,
+    }),
+  };
+
+  async run() {
+    const {
+      flags: {
+        sync,
+        "flow-url": flowUrl,
+        "integration-id": integrationIdFlag,
+        "include-execution-id": includeExecutionId,
+        "trigger-payload-file": triggerPayloadFilePath,
+      },
+    } = await this.parse(CniTestFlowCommand);
+
+    let triggerPayload: Record<string, string> = {};
+
+    if (triggerPayloadFilePath) {
+      if (await exists(triggerPayloadFilePath)) {
+        try {
+          triggerPayload = JSON.parse(
+            await fs.readFile(triggerPayloadFilePath, { encoding: "utf-8" }),
+          );
+        } catch (e) {
+          console.error("The provided trigger payload file contains malformed JSON");
+          throw e;
+        }
+      } else {
+        throw `No file found at ${triggerPayloadFilePath}. Please double check the --trigger-payload-file (-p) parameter.`;
+      }
+    }
+
+    let integrationId = integrationIdFlag;
+    let invokeUrl = flowUrl ?? "";
+
+    // Try to find an integrationId if we were not provided with an ID or
+    // direct invocation URL.
+    if (!invokeUrl && !integrationId) {
+      try {
+        const metadata = await getPrismMetadata();
+        integrationId = metadata.integrationId;
+      } catch (e) {
+        throw MISSING_PARAM_MESSAGE;
+      }
+
+      if (!integrationId) throw MISSING_PARAM_MESSAGE;
```

Comment:
```suggestion
      try {
        const metadata = await getPrismMetadata();
        integrationId = metadata.integrationId;
        
        if (!integrationId) {
          throw new Error(); // this will trigger the catch so you don't have to use a constant.
        }
      } catch (e) {
        throw "You must provide either a flow-url or an integration-id parameter."; // use handleError util 
      }
```